net.c

```diff
@@ -2152,7 +2153,7 @@ int _ifcfg_write(char *device, ifcfg_t *ifcfg)
 
   // set hostname, if requested
   if(config.net.sethostname) {
```

Comment:
so, it will work if I use ```SetHostname=1``` as bootparam. But I don't see if this flag is set in case of ```ifcfg=*=dhcp``` too